docs/userguide/datafiles.txt

```diff
@@ -89,19 +130,38 @@ the ``setup.py`` ``package_data`` list is updated before calling ``setup.py``.
 
 (Note: although the ``package_data`` argument was previously only available in
```

Comment:
This note is stale. Only Python 3.5 and later is supported and distutils is being merged into Setuptools.

Reply:
It seems like I'm not quite familiar with this part of setuptools. Should I just remove this note? I could dig deeper into it if this note should be updated.

Reply:
I'm just saying that any aspects relevant to Python 2.4 (or earlier than 3.5) can be re-written to assume Python 3.5.